src/components/AnnotationPopup/AnnotationPopup.js

```diff
@@ -61,11 +63,10 @@ const AnnotationPopup = () => {
     // calling this function will always rerender this component
     // because the position state always has a new object reference
     const setPopupPositionAndShow = () => {
-      if (popupRef.current) {
+      if (popupRef.current && popupItems && popupItems.length > 0) {
```

Comment:
prevent the "open" action from being dispatched because with if there are no items, it will show nothing in the DOM. so it might not make sense for it to have the state of "open"